server.py

```
@@ -68,6 +69,12 @@ async def websocket_endpoint(websocket: WebSocket):
                     # 5. BRAIN PHASE: Get Krishna's Hinglish response
                     krishna_text = llm_handler.get_krishna_response(user_text)
                     print(f"Krishna responds: {krishna_text}")
+                    latency_ms = (time.perf_counter() - start_time) * 1000
+                    try:
+                        analytics_db.log_conversation(session_id, user_text, krishna_text, latency_ms)
+                    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)